docs/english/concepts/message-sending.md

```diff
@@ -47,11 +49,11 @@ app.event('reaction_added', async ({ event, say }) => {
 
 You can have your app's messages stream in to replicate conventional AI chatbot behavior. This is done through three Web API methods:
 
-* [`chat.startStream`](/reference/methods/chat.startstream)
-* [`chat.appendStream`](/reference/methods/chat.appendstream)
-* [`chat.stopStream`](/reference/methods/chat.stopstream)
+- [`chat.startStream`](/reference/methods/chat.startstream)
+- [`chat.appendStream`](/reference/methods/chat.appendstream)
+- [`chat.stopStream`](/reference/methods/chat.stopstream)
```

Comment:
👁️‍🗨️ note: Using dashes instead of asterisk to avoid possible confusion with markdown in the listed item-